cfpb/consumerfinance.gov
# Update CCDB to latest code

See [release notes](https://github.com/cfpb/ccdb5-ui/releases/tag/v2.13.0) for details

## Additions

-


## Removals

-


## Changes

-


## How to test this PR

1. docker compose up
2. Load complaints via ccdb data pipeline
3. go to CCDB
4. Everything should look the same


## Screenshots


## Notes and todos

-


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Check the current browser support cutoff list](https://cfpb.github.io/consumerfinance.gov/browser-support#current-browser-support-metrics) for browsers that are advisable
to prioritize for testing.

#### Accessibility


#### Other
